src/main/scala/diplomaticobjectmodel/logicaltree/RocketLogicalTree.scala

```diff
@@ -0,0 +1,58 @@
+package diplomaticobjectmodel.logicaltree
+
+import freechips.rocketchip.config.Parameters
+import freechips.rocketchip.diplomacy.{LazyModule, ResourceBindingsMap, SimpleDevice}
+import freechips.rocketchip.diplomaticobjectmodel.model._
+import freechips.rocketchip.rocket.{DCacheParams, Frontend, ScratchpadSlavePort}
+import freechips.rocketchip.tile.{RocketTileParams, TileParams, XLen}
+
+
+class RocketLogicalTree(
+  device: SimpleDevice,
+  tileParams: TileParams,
+  rocketParams: RocketTileParams,
```

Comment:
`RocketTileParams extends TileParams`, so you should be able to use `rocketParams` anywhere you are using `tileParams` in the below code

Reply:
fixed